docs/rfcs/accepted/0003-svp_alloc/README.md

```diff
@@ -1,8 +1,10 @@
+# RFC 0003-SVP Alloc
```

Comment:
Why does the title differ from the feature name below?

Reply:
Fixed